service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/vm/AssignPetManagedIdentityAzureVmStep.java

```diff
@@ -15,16 +15,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class AssignManagedIdentityAzureVmStep implements Step {
+public class AssignPetManagedIdentityAzureVmStep implements Step {
```

Comment:
This step is specifically about the "pet" identity -- renamed to disambiguate with the action identities.